docs/ref/templates/builtins.txt

```diff
@@ -1074,7 +1077,12 @@ Argument            Outputs
 
 Sample usage::
 
-    {% templatetag openblock %} url 'entry_list' {% templatetag closeblock %}
+    {% templatetag openblock %}
+        url 'entry_list'
+        {% templatetag opencomment %}
+            Avoid template rendering via templatetag
+        {% templatetag closecomment %}
+    {% templatetag closeblock %}
```

Comment:
For this, I was thinking of simplifying the example, e.g.

```
The {% templatetag openblock %} characters open a block.
```

The reason is that it might be better to use `verbatim` for longer strings containing multiple characters (e.g. with an "open" followed by "close").

Reply:
Appreciate it! here Chris mentioned using 'isolated occurrence that doesn't have a matching pair'. Do you have any example for that also?